doc/LanguageClient.txt

```diff
@@ -63,6 +63,12 @@ accessed by regular quickfix/location list operations.
 To use the language server with Vim's formatting operator |gq|, set 'formatexpr': >
     set formatexpr=LanguageClient#textDocument_rangeFormatting_sync()
 <
+If you're using Neovim 0.4.0 or later, |LanguageClient#textDocument_hover()|
```

Comment:
I'd suggest move this section to function documentation as it is function specific.